Scripts/script.js

```diff
@@ -3,8 +3,8 @@ $(function() {
   (function projectCreator() {
 
     //variables for each project
-    var taskProject = new Project('taskOrganizerScreenshot.jpg', 'Task Organizer','2016-01-08', 'https://taskorganizer.herokuapp.com/');
-    var aimProject = new Project('aimBlasterScreenshot.jpg', 'Aim Blaster','2016-01-08', 'https://aimblaster.herokuapp.com/');
+    var taskProject = new Project('Assets/taskOrganizerScreenshot.jpg', 'Task Organizer','2016-01-08', 'https://taskorganizer.herokuapp.com/');
+    var aimProject = new Project('Assets/aimBlasterScreenshot.jpg', 'Aim Blaster','2016-01-08', 'https://aimblaster.herokuapp.com/');
 
```

Comment:
You may want to look at abstracting this out to a function that reads in external data sooner rather than later, much like our pair-programming blog project does with the `blogArticles.js` file. Hardcoding things like this becomes a maintenance nightmare quickly!